spec/lib/rbac/query_shared_resource_spec.rb

```diff
@@ -0,0 +1,50 @@
+describe RBAC::QuerySharedResource do
+  include_context "rbac_objects"
+
+  let(:options) do
+    { :verbs         => verbs,
+      :app_name      => app_name,
+      :resource_id   => resource_id1,
+      :resource_name => resource }
+  end
+  let(:roles) { [shared_role1, shared_role2] }
```

Comment:
I don't see `roles` being used anywhere in this file, this let can just go away I think.

Reply:
@eclarizio 
Roles are needed here because it is updated in the policy. A policy object has roles and groups.
When I build the policy double it needs the updated roles for this spec.